packages/twenty-front/src/modules/action-menu/actions/record-actions/constants/DefaultRecordActionsConfig.tsx

```diff
@@ -66,8 +66,8 @@ export const DEFAULT_RECORD_ACTIONS_CONFIG: Record<
     position: 0,
     isPinned: true,
     Icon: IconPlus,
-    shouldBeRegistered: ({ hasObjectReadOnlyPermission }) =>
-      !hasObjectReadOnlyPermission,
+    shouldBeRegistered: ({ objectPermissions }) =>
+      objectPermissions?.canUpdateObjectRecords === true,
```

Comment:
I'm confused by the logic:

If objectPermissions can be undefined, this means that no permission are provider, we should registered
If objectPermissions can't be undefined, why do we check for undefined case

Either way does not makes sense to me!

Reply:
or is it a loading temporary state? in this case, I would avoid calling this before we are ready